rednotebook/gui/main_window.py

```diff
@@ -663,48 +668,80 @@ def show_message(self, title, msg, msg_type):
         else:
             self.statusbar.show_message(title, msg, msg_type)
 
+    def update_undo_redo_buttons(self, _gobj=None, force_disable=False):
```

Comment:
The `_gobj` kwarg seems to be unused.

Reply:
It is, but this method is connected with a GTK signal handler, and it seems that it passes the GObject that sent the signal as the first argument. So without it, `force_disable` is given some unwanted object. I haven't read documentation about this, it was just trial and error, so maybe you know of a better way to deal with that.

Reply:
Done, but I've left `_gobject=None` so it's an optional argument - the method is also called from Python code (e.g. when switching from the HTML preview back to edit mode), and it doesn't seem right to pass a useless parameter in.

Reply:
Sounds good to me.